contract/contracts/NewInterfaces.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.6.0;
+
+contract DistributerInterface {
+    // TODO Add events
+
+    string name;
+    address tokenHolder;
+    mapping(address => address) tokenRefundDestinationList;
+
+    function balanceOf(address token) virtual external view returns (uint256) {}
+
+    function addToken() virtual external {}
```

Comment:
More than one creator may set tokens for a single distributor.
If so, how do you expect this to be managed?

Reply:
We use the same flow as above.